### ISSUE TITLE
Tried to call Protected function 'CastItemSpellOnFollowerAbility()'

### ISSUE THREAD

**Issue by Hentaya** [AcidWeb/RENovate#1]
Seems like a 'pure' error to me, not tainted by any other interactions. Maybe a newly protected function?
## Version: 1.5.0
Equipping a follower with an item:

AddOn 'RENovate' tried to call the protected function 'CastItemSpellOnFollowerAbility()'.
!BugGrabber\BugGrabber.lua:573: in function <!BugGrabber\BugGrabber.lua:573>
[C]: in function `CastItemSpellOnFollowerAbility'
...rrisonTemplates\Blizzard_GarrisonSharedTemplates.lua:59: in function `OnAccept'
FrameXML\StaticPopup.lua:4640: in function `StaticPopup_OnClick'
[string "*:OnClick"]:1: in function <[string "*:OnClick"]:1>

Locals:
nil

**Comment by AcidWeb** [comment]
This is only error? Fact that it shows RENovate as source it means nothing as taint detection in WoW is broken. Addon don't use or touch `CastItemSpellOnFollowerAbility()` or equipment slots at all.

But I will check it out. I presume that is error from live realm?

**Comment by Hentaya** [comment]
Yes, from live, US servers. 

I half-expected this wasn't a RENovate problem, as the addon has done nothing but perform excellently for me, and things I even SUSPECT are a problem, are fixed almost before I notice them.

I just didn't have the time (or the awareness through an alcholic buzz) to even do some simple checking. A bit more mentally clear this 'morning', I realized that I had applied equipment to two followers with no problem before this attempt.

Thank you for looking into it, and if you have any hints on tracing this sort of problem, I'd appreicate hearing them. :) I debug what I can, and I write when I get a chance, but when you spend all day churning out fixes to code and original code, you mostly just (or at least, *I* mostly just) wanna kill pixels when I get home. :)
